cardstack/src/components/TransactionConfirmationSheet/displays/RewardsRegisterDisplay.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { strings } from '../strings';
+import { SectionIconTitle } from './components/SectionIconTitle';
+import { PrepaidCardTransactionSection } from './components/sections/PrepaidCardTransactionSection';
+import { PayThisAmountSection } from './components/sections/PayThisAmountSection';
+import {
+  HorizontalDivider,
+  TransactionConfirmationDisplayProps,
+  IconProps,
+} from '@cardstack/components';
+import { RewardsRegisterData } from '@cardstack/types';
+
+interface RewardsRegisterDisplayProps
+  extends TransactionConfirmationDisplayProps {
+  data: RewardsRegisterData;
+}
+
+const rewardsIconProps = {
+  name: 'rewards',
+  size: 22,
+  color: 'teal',
+} as IconProps;
```

Comment:
```suggestion
const rewardsIconProps: IconProps = {
  name: 'rewards',
  size: 22,
  color: 'teal',
};
```
When we use `as` we are casting the value, so even if we add an `bleh` variable ts won't complain, when we 'assign'  like my suggestion, if we try to add `bleh` it will throw an error since `bleh` is not on `IconProps`

Reply:
Ah, yeah... thanks for the pointer.